### DISCUSSION TITLE
zeek networks.cfg not populated with $HOME_NET

### DISCUSSION THREAD

**Discussion by jimhranicky** [Security-Onion-Solutions/securityonion#6674] [Unsupported Versions] [answered]
Does SO2 by default not populate zeek's network.cfg with $HOME_NET ?
We were able to do so like so: 

    --- /opt/so/saltstack/default/salt/zeek/init.sls    2021-12-21 23:18:32.157531573 +0000
    +++ /opt/so/saltstack/local/salt/zeek/init.sls      2021-12-21 17:31:19.474191795 +0000
      @@ -127,6 +127,15 @@
         - group: 939
         - template: jinja
     
    +# Sync networks.cfg
    +zeeknetworkscfg:
    +  file.managed:
    +    - name: /opt/so/conf/zeek/networks.cfg
    +    - source: salt://zeek/files/networks.cfg
    +    - user: 937
    +    - group: 939
    +    - template: jinja
    +
     #zeekcleanscript:
     #  file.managed:
     #    - name: /usr/local/bin/zeek_clean
    @@ -209,6 +218,7 @@
           - /nsm/zeek/extracted:/nsm/zeek/extracted:rw
           - /opt/so/conf/zeek/local.zeek:/opt/zeek/share/zeek/site/local.zeek:ro
           - /opt/so/conf/zeek/node.cfg:/opt/zeek/etc/node.cfg:ro
    +      - /opt/so/conf/zeek/networks.cfg:/opt/zeek/etc/networks.cfg:ro
           - /opt/so/conf/zeek/zeekctl.cfg:/opt/zeek/etc/zeekctl.cfg:ro
           - /opt/so/conf/zeek/policy/securityonion:/opt/zeek/share/zeek/policy/securityonion:ro
           - /opt/so/conf/zeek/policy/custom:/opt/zeek/share/zeek/policy/custom:ro
    @@ -227,6 +237,7 @@
           - file: nodecfg
           - file: zeekctlcfg
           - file: zeekbpf
    +      - file: zeeknetworkscfg
       {% else %} {# if Zeek isn't enabled, then stop and remove the container #}
         - force: True
       {% endif %}

And adding our own file here: 

    /opt/so/saltstack/local/salt/zeek/files/networks.cfg

Is this something that could be added to the install script?

**Comment by dougburks** [comment, marked as answer]
Currently, HOME_NET is only used for Suricata:
https://docs.securityonion.net/en/2.3/homenet.html

We don't set this for Zeek currently because standard Zeek logging doesn't really require this value the same way that Suricata does. Is there something that you're trying to do with Zeek that requires this?

**Reply by jeffrylangvt** [reply]
We see an impact with this based on SSH connections.  The inbound/outbound direction for connections doesn't get populated in the SSH logs.  Which then requires for us a fairly complex destination.ip OR query of 10 large cidr networks to pull all inbound SSH connections.

**Reply by dougburks** [reply]
I've created https://github.com/Security-Onion-Solutions/securityonion/issues/6854 for this.
